python_src/src/lamp_py/performance_manager/gtfs_utils.py

```diff
@@ -16,9 +16,16 @@
 from lamp_py.aws.s3 import get_datetime_from_partition_path
 
 
+# boston tzinfo to be used with datetimes that require DST considerations.
+#
+# NOTE: do not use this as an argument for datetime.deatime constructors.
+# instead use BOSTON_TZ with a naive datetime. (https://pypi.org/project/pytz/)
+BOSTON_TZ = pytz.timezone("EST5EDT")
```

Comment:
made a local var here for the boston timezone since we use it a few times.

left a note on the constructor issue.